src/topp/FileInfo.cpp

```diff
@@ -376,19 +404,73 @@ class TOPPFileInfo :
     // Content statistics
     //-------------------------------------------------------------
     Map<String, int> meta_names;
-    if (in_type == FileTypes::FEATUREXML) //features
+
+	if (in_type == FileTypes::FASTA)
+	{
+		vector <FASTAFile::FASTAEntry> entries;
+		FASTAFile file;
+		vector <FASTAFile::FASTAEntry>::iterator loopiter;
+		vector <FASTAFile::FASTAEntry>::iterator iter;
+
+		map<char, int> aacids;
+		std::map<char, int>::iterator it;
+		int number_of_aacids = 0;
+		
+		mu.before();
+		//loading input
+		file.load(in, entries);
+
+		mu.after();
+		std::cout << "\n\nMemory usage while loading: " << mu << std::endl;
+
+		os << "Number of sequences: " << entries.size() << "\n"
+			<< "\n";
+	
+		for (loopiter = entries.begin(); loopiter != entries.end(); loopiter=std::next(loopiter))
+		{
+			iter = find_if(entries.begin(), loopiter, bind1st(mem_fun(&FASTAFile::FASTAEntry::HeaderMatches), &(*loopiter)));
+
+			if (iter != loopiter)
+			{
+				os << "WARNING: DUPLICATE SEQUENCE, NUMBER: " << std::distance(entries.begin(), loopiter) << ", ID: " << loopiter->identifier << " SAME AS NUMBER: " << std::distance(entries.begin(), iter) << ", ID: " << iter->identifier << "\n";
```

Comment:
Can you make the warnings (also below) in lower case please.
No reason to scream at the user.